railties/lib/rails/generators/test_unit/scaffold/scaffold_generator.rb

```diff
@@ -11,12 +11,19 @@ class ScaffoldGenerator < Base # :nodoc:
       class_option :api, type: :boolean,
                          desc: "Generates API functional tests"
 
+      class_option :system_tests, type: :string,
+                         desc: "Skip system test files"
+
       argument :attributes, type: :array, default: [], banner: "field:type field:type"
 
       def create_test_files
         template_file = options.api? ? "api_functional_test.rb" : "functional_test.rb"
         template template_file,
                  File.join("test/controllers", controller_class_path, "#{controller_file_name}_controller_test.rb")
+
+        unless options.api? || options[:system_tests].nil?
+          template "system_test.rb", File.join("test/system", class_path, "#{file_name.pluralize}_test.rb")
+        end
```

Comment:
I just tested this and there's a missing test case. This doesn't work for generating a scaffold and passing the skip system test option. 

`bin/rails g scaffold Tree title:string --skip-system-test`

In that case the system test file would still be generated because we're not checking at all for the `skip-system-test` option which is why I ended up changing the config file in the other PR. Because then otherwise we end up checking for 2 options for skipping system tests.